diff --git a/wrappers/python/setup.py b/wrappers/python/setup.py
index 356e6deb3..db0ae45f1 100644
--- a/wrappers/python/setup.py
+++ b/wrappers/python/setup.py
@@ -206,7 +206,7 @@ def buildKeywordDictionary(major_version_num=MAJOR_VERSION_NUM,
     # (not this DESTDIR-aware variant) for the same reason.
     openmm_lib_search_path = os.environ.get('DESTDIR', '') + openmm_lib_path

-    extra_compile_args=['-std=c++11']
+    extra_compile_args=['-std=c++11', '-D_GLIBCXX_USE_CXX11_ABI=1']
     extra_link_args=[]
     if platform.system() == "Windows":
         define_macros.append( ('WIN32', None) )
