diff --git a/platforms/cuda/sharedTarget/CMakeLists.txt b/platforms/cuda/sharedTarget/CMakeLists.txt
index d0430b85d..b731f58ff 100644
--- a/platforms/cuda/sharedTarget/CMakeLists.txt
+++ b/platforms/cuda/sharedTarget/CMakeLists.txt
@@ -22,4 +22,5 @@ ELSE (APPLE)
     SET_TARGET_PROPERTIES(${SHARED_TARGET} PROPERTIES LINK_FLAGS "${EXTRA_LINK_FLAGS}")
 ENDIF (APPLE)

+SET_TARGET_PROPERTIES(${SHARED_TARGET} PROPERTIES INSTALL_RPATH "$ORIGIN/..;$ORIGIN/../../../nvidia/cufft/lib;$ORIGIN/../../../nvidia/cuda_nvrtc/lib;")
 INSTALL_TARGETS(/lib/plugins RUNTIME_DIRECTORY /lib/plugins ${SHARED_TARGET})
