diff --git a/CMakeLists.txt b/CMakeLists.txt
index 603e797bc..5348937ba 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -355,7 +355,11 @@ ENDIF(OPENMM_BUILD_OPENCL_LIB)

 # HIP platform

-LIST(APPEND CMAKE_PREFIX_PATH $ENV{ROCM_PATH} /opt/rocm)
+IF(WIN32)
+    LIST(APPEND CMAKE_MODULE_PATH "C:/Program Files/AMD/ROCm/6.1/cmake")
+ELSE()
+    LIST(APPEND CMAKE_PREFIX_PATH $ENV{ROCM_PATH} /opt/rocm)
+ENDIF()
 FIND_PACKAGE(HIP CONFIG QUIET)
 IF(HIP_FOUND)
     SET(OPENMM_BUILD_HIP_LIB ON CACHE BOOL "Build OpenMMHIP library for AMD GPUs")
